src/main/java/org/threeten/extra/scale/MispInstant.java

```diff
@@ -0,0 +1,477 @@
+ * (https://nsgreg.nga.mil). The MISP Time System clock counts SI-Seconds since
+ * the Epoch of 1970-01-01T00:00:00.0Z (UTC). The MISP Time System is locked
+ * with TAI with a Leap Second offset of 8.000082 seconds. There are no leap
+ * seconds or other discontinuities.
+ *
+ * <p>
+ * This class may be used for instants in the far past and far future. Since
+ * some instants will be prior to 1970, it is not strictly an implementation of
+ * the MISP Time System. However it is equivalent to it since 1970.
+ *
+ * <h3>Implementation Requirements:</h3>
+ * This class is immutable and thread-safe.
+ * <p>
+ * This class must be treated as a value type. Do not synchronize, rely on the
+ * identity hash code or use the distinction between equals() and ==.
+ */
+public final class MispInstant
+        implements Comparable<MispInstant>, Serializable {
+    // does not implement Temporal as that would enable methods like
+    // Duration.between which gives the wrong answer due to lossy conversion
+
+    /**
+     * Constant for nanos per second.
+     */
+    private static final int NANOS_PER_SECOND = 1000000000;
+
+    /**
+     * Serialization version.
+     */
+    private static final long serialVersionUID = 5965510986576614924L;
+
+    /**
+     * Seconds part of the difference between TAI and MISP.
+     *
+     * <p>
+     * This is the number of seconds between 1958-01-01T00:00:00(TAI) and
+     * 1970-01-01T00:00.00(UTC).
+     */
+    private static final long TAI_OFFSET_SEC = 378691208;
+
+    /**
+     * Nanoseconds part of the difference between TAI and MISP.
+     *
+     * <p>
+     * This is the fractional part of the number of seconds between
+     * 1958-01-01T00:00:00(TAI) and 1970-01-01T00:00.00(UTC).
+     */
+    private static final long TAI_OFFSET_NANOS = 82000;
+
+    /**
+     * The number of seconds from the epoch of 1970-01-01T00:00:00(UTC).
+     */
+    private final long seconds;
+
+    /**
+     * The number of nanoseconds, later along the time-line, from the seconds
+     * field. This is always positive, and never exceeds 999,999,999.
+     */
+    private final int nanos;
+
+    /**
+     * Obtains an instance of {@code MispInstant} from the number of seconds
+     * from the MISP epoch of 1970-01-01T00:00:00(UTC) with a nanosecond
+     * fraction of second.
+     * <p>
+     * This method allows an arbitrary number of nanoseconds to be passed in.
+     * The factory will alter the values of the second and nanosecond in order
+     * to ensure that the stored nanosecond is in the range 0 to 999,999,999.
+     * For example, the following will result in the exactly the same instant:
+     * <pre>
+     *  MispInstant.ofMispSeconds(3, 1);
+     *  MispInstant.ofMispSeconds(4, -999999999);
+     *  MispInstant.ofMispSeconds(2, 1000000001);
+     * </pre>
+     *
+     * @param mispSeconds the number of seconds from the epoch of
+     * 1970-01-01T00:00:00(UTC)
```

Comment:
You can use a longer Javadoc line length to avoid wrapping most of these. When you do wrap, please indent by 2 spaces

Reply:
Have reworked things to avoid wrapping. Not sure how far is "too far", but it didn't look unreasonable.